modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java

```diff
@@ -54,12 +56,41 @@ public CTEntry toDTO(
 		return _toCTEntry(dtoConverterContext, ctEntry);
 	}
 
-	private Long _getSiteId(BaseModel<?> model) {
+	private String _getChangeTypeLabel(
+		Locale locale, com.liferay.change.tracking.model.CTEntry ctEntry,
+		BaseModel<?> model) {
+
+		if (model instanceof WorkflowedModel) {
+			WorkflowedModel workflowedModel = (WorkflowedModel)model;
+
+			if (workflowedModel.getStatus() ==
+					WorkflowConstants.STATUS_IN_TRASH) {
+
+				return _language.get(locale, "deleted");
+			}
+		}
+
+		if (ctEntry.getChangeType() == CTConstants.CT_CHANGE_TYPE_ADDITION) {
+			return _language.get(locale, "added");
+		}
+
+		return _language.get(locale, "modified");
+	}
```

Comment:
@noornajj I believe you accidentally added this change back when you resolved the conflicts. It was removed in another change I sent yesterday.